charts/logging-operator-logging/templates/logging.yaml

```diff
@@ -54,4 +54,9 @@ spec:
   {{- else }}
   fluentbit: {}
   {{- end }}
-
+{{- if .Values.nodeAgents }}
+{{ toYaml .Values.nodeAgents | indent 4 }}
+    {{- end }}
+{{- else }}
+    nodeAgents: {}
+{{- end }}
```

Comment:
Wouldn't we be missing the `nodeAgents:` field for this? Would it be better to do the following?

```yaml
{{- with .Values.nodeAgents }}
nodeAgents:
  {{- toYaml . | nindent 4 }}
{{- end }}
```